task1/index.js

```diff
@@ -0,0 +1,117 @@
+const notes = [
+    {
+        id: 1,
+        title: "Recipe",
+        description: "Ingredients include 2 eggs...",
+        pagesCount: 2,
+        isMarked: false,
+        access: []
+    },
+    {
+        id: 2,
+        title: "Workouts",
+        description: "3 sets of squats...",
+        pagesCount: 1,
+        isMarked: true,
+        access: []
+    },
+    {
+        id: 3,
+        title: "Passwords",
+        description: "VISA ...",
+        pagesCount: 6,
+        isMarked: true,
+        access: []
+    },
+    {
+        id: 4,
+        title: "To Do 2021",
+        description: "1. Learn JS...",
+        pagesCount: 3,
+        isMarked: false,
+        access: []
+    }
+]
+
+// MAP POLYFIL
+Array.prototype.myMap = function (callback) {
+    let result = []
+    for (let i = 0; i < this.length; i++) {
+        result.push(callback(this[i], i, this))
+    }
+    return result
+}
+
+
+// MAP
+Array.prototype.myMap = function (callback) {
+    let result = []
+    for (let i = 0; i < this.length; i++) {
+        result.push(callback(this[i], i, this))
+    }
+    return result
+}
+
+let result = notes.myMap((item) => {
```

Comment:
Why `let`? Where do you plan to change this variable?
`item` it's not a good name, replace with something more clear(Example: note)
Rename result to something more clear: `const updatedNotes = `